test/integration/middleware_test.rb

```diff
@@ -39,7 +39,6 @@ class MiddlewareIntegrationTest < ActionDispatch::IntegrationTest
       test 'it is added to the Content-Security-Policy as well' do
         logout_admin
         visit '/environments'
-        assert page.has_selector? 'input[name="login[password]"]'
```

Comment:
instead of two tests that do the same, how about moving this line to `middleware_js_test.rb`?

Reply:
trying to add this line to the test:
`assert page.response_headers['Content-Security-Policy'].include?(@webpack_url)`
and got this error on that test:
```
TypeError: undefined is not a function (evaluating '$(document)')
TypeError: undefined is not a function (evaluating '$(document)')
    at http://127.0.0.1:42973/assets/application-84f5c7fd6ba642ee6c40ab366cf45edd21ac6c531ec066b7101f46f7cffbc04c.js:17
ReferenceError: Can't find variable: $
ReferenceError: Can't find variable: $
    at http://127.0.0.1:42973/users/login:47 in global code
ReferenceError: Can't find variable: $
ReferenceError: Can't find variable: $
    at http://127.0.0.1:42973/users/login:54 in global code
    test/integration/middleware_js_test.rb:41:in `block (3 levels) in <class:MiddlewareJSTest>'
```

not using any jQuery on the login page though..